packages/sit-onyx/src/components/OnyxDataGrid/features/sorting/sorting.ts

```diff
@@ -53,7 +53,7 @@ export const useDataGridSorting = createFeature(
       return !config || config?.[col]?.enabled === true;
     });
 
-    const locale = injectI18n()?.locale;
+    const locale = computed(() => injectI18n()?.locale.value ?? "en-us");
```

Comment:
The default / fallback value is handeled by i18n so the locale is always defined:
![image](https://github.com/user-attachments/assets/fe0aa349-6e1c-4aee-9498-a29ba6dff0cb)

You can just use it with:
```ts
const { locale } = injectI18n();
```